pkgs/tools/typesetting/asciidoctor/gemset.nix

```diff
@@ -421,3 +450,4 @@
     version = "1.5.1";
   };
 }
+
```

Comment:
Please remove this extra final new line.

Reply:
done.